R/render-plot.R

```diff
@@ -333,10 +335,34 @@ custom_print.ggplot <- function(x) {
   gtable <- ggplot2::ggplot_gtable(build)
   grid::grid.draw(gtable)
 
-  structure(list(
+  res <- list(
     build = build,
     gtable = gtable
-  ), class = "ggplot_build_gtable")
+  )
+  class(res) <- "ggplot_build_gtable"
+
+  # ggplot2::get_alt_text() was added in v3.3.4
+  # https://github.com/tidyverse/ggplot2/pull/4482
+  get_alt <- asNamespace("ggplot2")$get_alt_text
```

Comment:
`asNamespace()` has always felt a bit squishy to me; `getNamespace()` is slightly clearer:

```suggestion
  get_alt <- getNamespace("ggplot2")$get_alt_text
```